handler/src/main/java/io/netty/handler/internal/svm/ssl/Java9SslEngineSubstitution.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.internal.svm.ssl;
+
+import io.netty.handler.ssl.JdkApplicationProtocolNegotiator;
+
+import javax.net.ssl.SSLEngine;
+
+import com.oracle.svm.core.annotate.Alias;
+import com.oracle.svm.core.annotate.TargetClass;
+import com.oracle.svm.core.jdk.JDK11OrLater;
+
+@TargetClass(className = "io.netty.handler.ssl.Java9SslEngine", onlyWith = JDK11OrLater.class)
```

Comment:
shouldn't that be JDK9OrLater ?

Reply:
You right, but when we wrote this, the given Graal SVM library had only JDK 8 / 10 or earlier JDK 11 / 13 or later  respectively. It seems that this part of SVM is moving quite fast, looks different now:

https://github.com/oracle/graal/tree/release/graal-vm/19.3/substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk

That's kinda though to answer and affects the Java driver for Neo4j of course too. 🤔

Reply:
Updated my comment above.